fitbenchmarking/core/fitting_benchmarking.py

```diff
@@ -111,27 +113,29 @@ def loop_over_benchmark_problems(problem_group, options):
 
     name_index = {key: 0 for key in name_count}
     LOGGER.info('Running problems')
-    for i, (fname, problem) in enumerate(problems):
-        # Make the name unique
-        if name_count[problem.name] > 1:
-            name_index[problem.name] += 1
-            problem.name += f' {name_index[problem.name]}'
-
-        info_str = f" Running data from: {os.path.basename(fname)} " + \
-                   f"{i + 1}/{len(problem_group)}"
-        LOGGER.info('\n%s', '#' * (len(info_str) + 1))
-        LOGGER.info(info_str)
-        LOGGER.info('#' * (len(info_str) + 1))
-
-        ##############################
-        # Loops over starting values #
-        ##############################
-        problem_results, problem_fails, \
-            unselected_minimizers = \
-            loop_over_starting_values(
-                problem, options, grabbed_output)
-        results.extend(problem_results)
-        failed_problems.extend(problem_fails)
+    pb = tqdm(problems,colour='green',leave=True)
+    with logging_redirect_tqdm():
```

Comment:
Specifying the logger seems to get it working!
```suggestion
    with logging_redirect_tqdm(loggers=[LOGGER]):
```

Reply:
It does, cheers however it is printing the progress bar on every line now. 

![image](https://user-images.githubusercontent.com/83226114/193245051-817a8d6d-2cfe-47a3-bf30-39911a24df82.png).

I edited the "tqdm()" so "leave" is = "false" and it hasn't changed anything.  

![image](https://user-images.githubusercontent.com/83226114/193245162-7eaf8843-5405-4cb0-b662-b2c1cd11713e.png)

I'll investigate how to prevent this from happen and keep the progress bar to the bottom of command line output.